sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessor.cs

```diff
@@ -71,7 +71,7 @@ public class ServiceBusProcessor : IAsyncDisposable
         /// Gets the ID to identify this processor. This can be used to correlate logs and exceptions.
         /// </summary>
         /// <remarks>Every new processor has a unique ID.</remarks>
-        internal string Identifier { get; }
+        public string Identifier { get; }
```

Comment:
Should we consider plumbing the Identifier through to the receivers that the processor uses? Most of the logs generated when using a processor will come from various receivers. See https://github.com/Azure/azure-sdk-for-net/issues/28360. I can see us doing this only if someone sets the identifier (to avoid breaking anyone). Another option would be to add an additional configuration option to indicate whether the processor identifier should be used for all clients that the processor uses.
/cc @jsquire

Reply:
I believe that we should.   In Event Hubs, we use the processor's identifier as the basis for the consumers for each partition with the pattern "P{partitionId}-{processor.identifier}";  I think we should consider something similar here.   

@JoshLove-msft:  Does the Service Bus processor have any kind of pattern where it uses multiple receivers outside of sessions where we'd need a pattern?   Should we consider something like "{processor.identifier}-{sessionId}" or similar for sessions?

Reply:
Currently, we don't have any sort of pattern like that. I think this would work though and it would address the concern I mentioned in the issue about customers potentially wanting separate identifiers for the individual receivers, while still allowing them to pattern match on the common processor prefix.

Reply:
@JoshLove-msft  that makes sense to me for the `SessionReceiverManager` receivers, but for the `ReceiverManager`s would those just propagate the processor Id and ignore the duplication issue?

Reply:
Good point, I think for non-session we can just append "-Receiver"